_tests/integration/reactnative_expo_test.go

```diff
@@ -33,7 +33,26 @@ func TestReactNativeExpo(t *testing.T) {
 		result, err := fileutil.ReadStringFromFile(scanResultPth)
 		require.NoError(t, err)
 
-		validateConfigExpectation(t, "Managed Expo Workflow", strings.TrimSpace(managedExpoResultsYML), strings.TrimSpace(result), managedExpoVersions...)
```

Comment:
Versions are already applied to _managedExpoResultsYML_, so the parameter _managedExpoVersions_ is not needed.